blocks/donation-form/__payment-logo/_type/donation-form__payment-logo_type_maestro.css

```diff
@@ -0,0 +1,4 @@
+.donation-form__payment-logo_type_maestro {
```

Comment:
modificator without block or element

Reply:
There is no common rules for logos. And creating empty files is undesirable.